Tests/TransferMethod/SelectTransferMethodTypePresenterTests.swift

```diff
@@ -92,7 +115,7 @@ class SelectTransferMethodTypePresenterTests: XCTestCase {
                       "The navigateToAddTransferMethodControllerPerformed should be performed")
 
         XCTAssertEqual(presenter.countryCurrencyCount, 2, "The countryCurrencyCount should be 2")
-        XCTAssertEqual(presenter.transferMethodTypesCount, 3, "The transferMethodTypesCount should be 3")
+        XCTAssertEqual(presenter.transferMethodTypesCount, 1, "The transferMethodTypesCount should be 3")
```

Comment:
Update the comment as well to 1 instead of 3

Reply:
Sure